src/Mollie/WC/Gateway/Abstract.php

```diff
@@ -1749,22 +1751,24 @@ public function getReturnUrl (WC_Order $order)
 	    if ( strpos( $lang_url, '/?' ) !== false ) {
 		    $lang_url_params = substr( $lang_url, strpos( $lang_url, "/?" ) + 2 );
 		    $return_url = $return_url . '&' . $lang_url_params;
-	    } else {
-		    $return_url = str_replace( $site_url, $lang_url, $return_url );
 	    }
 
-	    if ( version_compare( WC_VERSION, '3.0', '<' ) ) {
-		    Mollie_WC_Plugin::debug( $this->id . ': Order ' . $order->id . ' returnUrl: ' . $return_url, true );
-	    } else {
-		    Mollie_WC_Plugin::debug( $this->id . ': Order ' . $order->get_id() . ' returnUrl: ' . $return_url, true );
-	    }
+        if (version_compare(WC_VERSION, '3.0', '<')) {
+            debug($this->id . ': Order ' . $order->id . ' returnUrl: ' . $return_url, true);
+        } else {
+            debug($this->id . ': Order ' . $order->get_id() . ' returnUrl: ' . $return_url, true);
+        }
```

Comment:
I would wrap this occurrencies into a function, something like 

```
$orderId = (int) orderId($order);
```

The function than can be something like: 

```
function orderId(WC_Order $order) {
  return version_compare(WC_VERSION, '3.0', '<')
    ? $order->id
    : $order->get_id();
}
```

This will make more simple to refactor the code when we'll drop support for WC < 3.0, just change one like of code from `orderId($order)` to `$order->get_id()`.